jabref-template/listrefs.begin.layout

```diff
@@ -406,27 +406,44 @@ p.infolinks { margin: 0.3em 0em 0em 0em; padding: 0px; }
 <body>
     <!--#include file="header.include" -->
 
+    <div class="bs-header">
+      <div class="container">
+        <br />
+        <h2> Publications on and with <abbr>deal.II</abbr>
+        </h2>
+
+        <p>
+          This page is an attempt at keeping track of all publications
+          that use and describe deal.II. Keeping it up to date is good
+          for our users (who can see what others have already done),
+          but is also important for keeping our funding agencies
+          appraised how widely used deal.II actually is.
+        </p>
+        
+        <p>
+          If you have published anything,
+          including Diploma, Masters or PhD theses, please
+          <a href="https://github.com/dealii/publication-list/blob/master/README.md">let
+          us know about it</a>!
+        </p>
+      </div>
+    </div>
+
     <div id="content" class="container">
-    <div class="row">
-          <div class="well">
-            <h2>Publications</h2>
-	    <p>
-	      Number of publications using deal.II, see <a href="#list">below</a> for details.
-	    </p>
+      <div class="row">
+        <div class="col-xs-12 col-md-9">
+          <div class="well" align="center">
 	    <div id="plot" style="width:600px"></div>
 	    <div id="n_publications"></div>
 	  </div>
-    </div>
-
+        </div>
+      </div>
```

Comment:
There are three new `<div>`'s in this block, but only two `</div>`'s. Is this perhaps the cause of the build failure?